releasenotes/notes/celery-flatten-context-dicts-bc5c33b72bd72ac2.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    celery: Fixes issue #4771 where context tags containing dictionaries are sent as raw strings instead of being flattened into separate tags.
```

Comment:
I think I'd do this as a `features:` since it is (IMHO) not fixing broken behavior, but improving/enhancing what is there.

what do you think?

Reply:
I'm okay to change it to `features`. Marked it as `fixes` because for our use case, the dictionary as a raw string wasn't very useful other than as a data point.

Will change to feature I think